### ISSUE TITLE
RxNorm Extension and Missing Ingredients

### ISSUE THREAD

**Issue by ericaVoss** [OHDSI/Vocabulary-v5.0#126]
I'm seeing RxNorm Extensions missing some ingredient associations.

```{SQL}
SELECT c.CONCEPT_ID, c.CONCEPT_NAME, c1.CONCEPT_ID, c1.CONCEPT_NAME
FROM CONCEPT c
	JOIN CONCEPT_ANCESTOR ca
		ON c.CONCEPT_ID = ca.DESCENDANT_CONCEPT_ID
	JOIN CONCEPT c1
		ON c1.CONCEPT_ID = ca.ANCESTOR_CONCEPT_ID
		AND c1.CONCEPT_CLASS_ID = 'Ingredient'
WHERE c.CONCEPT_ID IN ( 43293656,43277463,43277461)
```

```{TEXT}
CONCEPT_ID	CONCEPT_NAME	CONCEPT_ID	CONCEPT_NAME
43277461	Desogestrel 0.15 MG / Ethinyl Estradiol 0.03 MG [Madeline]	1588000	Desogestrel
43293656	Ethinyl Estradiol 0.03 MG / drospirenone 3 MG [Petibelle]	1512674	drospirenone
43277463	Ethinyl Estradiol 0.03 MG / drospirenone 3 MG [Isabelle]	1512674	drospirenone
```

Here are a few missing their `	1549786-	Ethinyl Estradiol` map.

**Comment by dimshitc** [comment]
Yep, "43277461	Desogestrel 0.15 MG / Ethinyl Estradiol 0.03 MG [Madeline]" was linked to "1549948 Ethinyl Estradiol 0.000833 MG/HR", and then this concept was deprecated by RxNorm.
Anyway, in the next release it will be alright, our development tables already have links to both ingredients.

**Comment by ericaVoss** [comment]
Okay, sounds good!  I'm just making a note that this still is a problem in 20170503 but will check back in a future version.

**Comment by ericaVoss** [comment]
For these drugs, this is fixed in 20170920.
